src/components/Plants/SpeciesByPlotChart.tsx

```diff
@@ -26,12 +29,17 @@ export default function SpeciesByPlotChart({ plots }: Props): JSX.Element {
   const onChangePlot = (newValue: string) => {
     if (plots) {
       setSelectedPlot(plots.find((p) => p.fullName === newValue));
+      updatePlotPreferences(newValue);
     }
   };
 
   React.useEffect(() => {
-    setSelectedPlot(undefined);
-  }, [plots]);
+    if (lastPlot) {
+      setSelectedPlot(plots?.find((p) => p.fullName === lastPlot));
+    } else {
+      setSelectedPlot(undefined);
```

Comment:
default to the first plot here or if we don't find the lastPlot in the 'if' above.
also should we be checking for id instead of fullName?
```
  if (!plots?.length) {
    return;
  }
  const plot = plots.find(p => p.id === lastPlot);
  setSelectedPlot(plot || plots[0]);
```